.github/workflows/common.yml

```diff
@@ -28,7 +28,14 @@ jobs:
   unitTest:
     name: Unit test
     runs-on: ubuntu-18.04
+    services:
+      rabbitmq:
```

Comment:
rabbitmq shouldn't be part of a common job.

Reply:
Yep, this was my misunderstanding of the exclusion of component tests from common.  I'll get rid of it when I exclude the tests here.